api/db/populate-messages.js

```diff
@@ -27,12 +27,12 @@ const createMessages = async (verbose) => {
   const messagesArray = [];
 
   for (const friendship of friendshipsArray) {
-    const messagesCount = Math.floor(Math.random() * 10);
+    const messagesCount = Math.floor(Math.random() * 25);
 
     for (let i = 0; i < messagesCount; i++) {
       const user =
         Math.random() < 0.5 ? friendship.requestant : friendship.receivant;
-      const content = getArticle();
+      const content = getParagraph();
```

Comment:
Maybe we could pass a `Math.ceil(Math.random() * 3)` parameter to this `getParagraph()` function?

Reply:
Oops, this comment doesn't make sense, please disregard it